vnext/src/RNTester/js/RNTesterList.windows.ts

```diff
@@ -238,7 +238,7 @@ const APIExamples: Array<IRNTesterExample> = [
   },*/
 ];
 
-const Modules: {[key: string]: IRNTesterModule} = {};
+const Modules: { [key: string]: IRNTesterModule } = {};
```

Comment:
>{ [key: string]: IRNTesterModule } [](start = 15, length = 34)

The CI loop is unhappy about this change.  Are you using your own prettier formatter tool?

I've been getting in the habit of running yarn lint:fix from the vnext directory before pushing changes.

Reply:
I did that for the last commit here 😶 Curious why this wasn't picked up.

VSCode is using the wrong prettier config on save on my machine right now. I should probably spend some time trying to fix that.